### ISSUE TITLE
Automatic localization for messages

### ISSUE THREAD

**Issue by danielhers** [data-for-change/anyway#94]
Currently messages are written as Hebrew strings hard-coded in the source code.
They should be written in a .po file and translated.

**Comment by esegal** [comment]
Which i18n framework should be used? 
I saw https://pythonhosted.org/Flask-Babel/ for server side and http://l20n.org/ for client side.
Do we have a preference?

**Comment by danielhers** [comment]
`Flask-Babel` looks good. I think it also works for the templates: http://stackoverflow.com/questions/9515156/flask-babel-how-to-use-translation-in-jinja-template-file

**Comment by esegal** [comment]
I started working on integrating Flask-Babel. I'll soon push the changes, and then we need to start translating everything to English.

**Comment by omerxx** [comment]
I think an English translation already exists, ask @galraij

**Comment by danielhers** [comment]
Right, for the fields but not the messages, I think. Both can be integrated
into .po files.

**Comment by esegal** [comment]
To start, I've integrated Flask-Babel (https://github.com/hasadna/anyway/pull/438). This will allow replacing strings with gettext('Message') function, to get the translated message according to the current request's chosen locale.

**Comment by danielhers** [comment]
From #310, here is the translation file for the data fields: https://docs.google.com/spreadsheets/d/1TxRQukcLzfno8kG-6eH_XEMrcuk2h8PNJOmY5IZUEzg/edit?usp=sharing

**Comment by danielhers** [comment]
See #438 for instructions on how to write translations.

**Comment by danielhers** [comment]
Accident details field translations:

[Anyway ACC code book translation into english.xlsx](https://github.com/hasadna/anyway/files/15503/Anyway.ACC.code.book.translation.into.english.xlsx)
